config/app/packages/webpack_encore.yaml

```diff
@@ -9,3 +9,9 @@ when@prod:
         builds:
             # The path where Encore is building the assets - i.e. Encore.setOutputPath()
             app: '%kernel.project_dir%/public/assets/app/build'
+
+when@test:
+    webpack_encore:
+        builds:
+            # The path where Encore is building the assets - i.e. Encore.setOutputPath()
+            app: '%kernel.project_dir%/public/assets/app/build_dev'
```

Comment:
Should we use prod assets in test env to make it a bit faster?

Reply:
We could - but that would requires us to have prod assets ready - which is not necessary at the time of testing.

Reply:
We can do - but it is not straightforward.
The idea was to test before doing anything else during deployment so that if it fails, it fails as early as possible. This would mean moving asset build to the beginning of the deploy process, and having to split it into two parts - as right now asset build and copy to prod are bundled together.
It does not seem to me this brings significant performance boost as it is now, but if you'd like, I can make those changes.

Reply:
No need, it was just an idea.

Maybe letter, when and if we have a significant number of tests and if it proves to be a bottleneck.